skimage/filters/__init__.py

```diff
@@ -59,4 +60,5 @@
            'threshold_multiotsu',
            'apply_hysteresis_threshold',
            'rank',
-           'unsharp_mask']
+           'unsharp_mask',
+           'get_windownd']
```

Comment:
Can we just call it `get_window`? Long term, it should be the assumption in scikit-image that functions are nd. ;)

Reply:
I was worried about possible confusion with `scipy.signal.get_window`, but if this is not a concern, I agree that would be preferred